plugins/Marketplace/vue/src/PluginList/CTAContainer.vue

```diff
@@ -0,0 +1,231 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <template v-if="isSuperUser">
+    <div v-if="plugin.isMissingLicense"
+         class="alert alert-danger alert-no-background">
+      {{ translate('Marketplace_LicenseMissing') }}
+      <span
+        style="white-space:nowrap"
+      >(<a tabindex="7"
+           class="plugin-details"
+           href="#"
+           v-plugin-name="{ pluginName: plugin.name }"
+           :title="translate('General_MoreDetails')"
+      >{{ translate('General_Help') }}</a>)</span>
```

Comment:
The whole <a...> element was previously a macro that could be reused, here we unpacked it back to 7 or 8 instances of the same or nearly the same. I reckon it should be a tiny `LinkButton` or `CTAButton` Vue component (similar to the `DownloadButton` below) with a few props that can get passed from here like they did previously using the macro.

Reply:
Happy to hear other opinions here but having worked with the cards before during the redesign it made sense to avoid the repetitivness.

Reply:
Extracted parts into a new `HelpLink` and `MoreDetailsButton`. For the other links, that are not used multiple times, I created small methods to remove the href generation from the template.